packages/shared/src/utils/validation/declarative-validation-rules.ts

```diff
@@ -27,7 +27,7 @@ export const validLetterSymbol = (value: string) => /^[A-Za-z]+([a-zA-Z.' -])*[a
 export const validName = (value: string) => /^(?!.*\s{2,})[\p{L}\s'.-]{2,50}$/u.test(value);
 export const validLength = (value = '', options: TOptions) =>
     (options.min ? value.length >= options.min : true) && (options.max ? value.length <= options.max : true);
-export const validPassword = (value: string) => /(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]+/.test(value);
+export const validPassword = (value: string) => /^(?=.*[a-z])(?=.*\d)(?=.*[A-Z])[ -~]{8,25}$/.test(value);
```

Comment:
FE and BE validation for password was different. In FE non -english letters were not validated and hence able to proceed for setting the password. This throws BE invalidation error.
Here we are setting the password validation regex same as BE